lib/vyasa_web/components/contexts/read.ex

```diff
@@ -273,11 +357,13 @@ defmodule VyasaWeb.Context.Read do
 
   # fallthrough
   def init_drafting_context(%Socket{} = socket) do
+        #%Sheaf{marks: [Mark.get_draft_mark()]}
```

Comment:
i am thinking of initializing draft reflector for first splash, this is to help with preview of binding etc

Reply:
related to the question of "how to deal with first splash" 

if the sangh session info is not there yet, then should we be previewing that? (e.g. preview the mark)... 


the fix for this will come in a separate PR